jscomp/build_tests/super_errors/expected/arity_mismatch.res.expected

```diff
@@ -6,5 +6,5 @@
   [1;31m2[0m [2m│[0m let makeVariables = [1;31mmakeVar[0m(.~f=f => f)
   3 [2m│[0m 
 
-  This uncurried function has type (. ~f: 'a => 'a, unit) => int
+  This uncurried function has type (~f: 'a => 'a, unit) => int
```

Comment:
In a separate PR, we should change "This uncurried function has ..." to "This function has ...".